src/Ticket.php

```diff
@@ -1250,11 +1250,15 @@ public function prepareInputForUpdate($input)
             list($dateField, $slaField) = SLA::getFieldNames($slmType);
             if (isset($input[$slaField]) && ($input[$slaField] > 0)) {
                 $manual_slas_id[$slmType] = $input[$slaField];
+            } elseif (!isset($input[$slaField])) {
+                $input[$slaField] = $this->fields[$slaField];
```

Comment:
IMHO, to help people to understand why these values are forced, added code should be moved in `fillInputForBusinessRules` and a comment should be added.